baseagent.py

```diff
@@ -1102,7 +1102,7 @@ def service_quit( self, msg='' ):
         metadata[ 'status' ] = 'finished'
         metadata[ 'error-message' ] = 'null'
 
-        asyncio.ensure_future( self.schedule_message( self.holon, metadata=metadata ) )
+        await self.schedule_message( self.holon, metadata=metadata )
```

Comment:
I suppose `asyncio.ensure_future` was an attempt to replace `await`, however, it looks like it would still invoke `schedule_message` in `async` manner and not `sync`